frontend/src/features/utils.js

```diff
@@ -39,4 +39,12 @@ export const convertPaginationToParamsOffset = (pagination) => {
     }
 
     return result
-} 
\ No newline at end of file
+} 
+
+//Removes the slashes('/') present at the end of url
+export const getValidURL = (url) => {
+    while(url?.length > 0 && url[url.length-1] === "/"){
+        url = url.slice(0,url.length-1)
+    }
+    return url;
+}
```

Comment:
```suggestion
// removes the trailing slashes from given url
export const cleanURL = (url) => {
  return url.replace(/\/+$/, '');
}
```

Rename the function name to `cleanURL`.